core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -135,6 +136,7 @@ object KafkaConfig {
       .define(ServerConfigs.REQUEST_TIMEOUT_MS_CONFIG, INT, ServerConfigs.REQUEST_TIMEOUT_MS_DEFAULT, HIGH, ServerConfigs.REQUEST_TIMEOUT_MS_DOC)
       .define(ServerConfigs.SOCKET_CONNECTION_SETUP_TIMEOUT_MS_CONFIG, LONG, ServerConfigs.DEFAULT_SOCKET_CONNECTION_SETUP_TIMEOUT_MS, MEDIUM, ServerConfigs.SOCKET_CONNECTION_SETUP_TIMEOUT_MS_DOC)
       .define(ServerConfigs.SOCKET_CONNECTION_SETUP_TIMEOUT_MAX_MS_CONFIG, LONG, ServerConfigs.SOCKET_CONNECTION_SETUP_TIMEOUT_MAX_MS, MEDIUM, ServerConfigs.SOCKET_CONNECTION_SETUP_TIMEOUT_MAX_MS_DOC)
+      .define(ServerConfigs.METADATA_RECOVERY_STRATEGY_CONFIG, STRING, ServerConfigs.DEFAULT_METADATA_RECOVERY_STRATEGY, LOW, ServerConfigs.METADATA_RECOVERY_STRATEGY_DOC)
```

Comment:
Which clients in the broker do we want to rebootstrap? Don't remember this change from the KIP.

Reply:
You're right, it's not needed here. Removed